### PR TITLE
feat: qrcode tt logo

### PR DESCRIPTION
**Pull request by simboonlong**
### update
- qrcode with tt logo instead
- related to https://github.com/TradeTrust/tradetrust-website/pull/248

### preview
Screenshot 2020-10-13 at 10 18 48 AM

### CODE REVIEW

File: src/components/DocumentUtility/DocumentUtility.tsx [hunk 1]
```diff
@@ -23,27 +22,17 @@ export const DocumentUtilityUnStyled = ({
...
 }: DocumentUtilityProps) => {
-  const fileName = getData(document).name;
+  const fileName = getData(document)?.name ?? "Untitled";
   const qrcodeUrl = getData(document)?.links?.self?.href ?? "";
-  const logoUrl = getData(document)?.logo;
-  const [imageSettings, setImageSettings] = useState<ImageSettings>();
+  const imageSettings: ImageSettings = {
+    src: `/static/images/logo-qrcode.png`,
+    x: undefined,
+    y: undefined,
```
Comment (yehjxraymond) [inline src/components/DocumentUtility/DocumentUtility.tsx]: how come need undef here? Cannot just dont describe it?

Comment (simboonlong) [reply src/components/DocumentUtility/DocumentUtility.tsx]: actually its optional, let me omit them

File: src/components/DocumentUtility/DocumentUtility.tsx [hunk 2]
```diff
@@ -35,27 +34,15 @@ export const DocumentUtilityUnStyled = ({
 }: DocumentUtilityProps) => {
   // Extending document data to account for undefined metadata in OA schema
   const documentWithMetadata = getData<WrappedDocument<DocumentWithAdditionalMetadata>>(document);
-  const fileName = documentWithMetadata.name;
+  const fileName = documentWithMetadata.name ?? "Untitled";
```
Comment (yehjxraymond) [inline src/components/DocumentUtility/DocumentUtility.tsx:37]:
```suggestion
  const fileName = documentWithMetadata.name || "Untitled";
```

**Review by yehjxraymond — APPROVED** [review]
lgtm

**Pull request merged**